MIGRATING.md

```diff
@@ -0,0 +1,84 @@
+# Okta Signin Widget migration guide
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
+
+## Migrating from 2.x to 3.x
+
+### Consolidated css files to single one
+
+In version 2.x we had two separate css files to import: `okta-sign-in.min.css` and `okta-theme.css`. We moved to have one single css file which is still called `okta-sign-in.min.css`.
+
+Depending on your configuration, this is what you should change:
+
+- If you were using CDN links for the css, you will need to change the `okta-sign-in.min.css` link and remove the `okta-theme.css` link.
+
+Example:
+```html
+<!-- BEFORE -->
+<link
+  href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.14.0/css/okta-sign-in.min.css"
```

Comment:
Curious if you think we should be more generic with the version, and say `/okta-signin-widget/{version}/css/..`?

Reply:
Since it's an actual example and I wanted to explicitly show passing from 2.x to 3.x, I opted for showing a real case.